seonshine_frontend/src/pages/signUp/ChooseUserType/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import FoodBankIcon from '@mui/icons-material/FoodBank';
+import { Box, Button, Stack, Typography, useMediaQuery } from '@mui/material';
+
+import { EnterUserTypePropsType } from '../types';
+
+const ChooseUserType = ({ handleSubmitUserType }: EnterUserTypePropsType) => {
+  const isMobile = useMediaQuery('(max-width:600px)');
+  return (
+    <Stack
+      alignItems="center"
+      justifyContent="center"
+      className="min-h-[80vh]"
+    >
+      <Stack
+        direction="column"
+        gap="24px"
+        alignItems="center"
+        className="w-full p-24 bg-white rounded-lg shadow-md md:w-240 xl:w-240 max-w-screen"
+      >
+        <Typography
+          variant="heading2"
+          component="h2"
+          className="text-center"
+        >
+          Sign Up
+        </Typography>
+        <Typography
+          variant="bodyS"
+          className="text-center min-w-max"
+        >
+          Choose your type of user
+        </Typography>
+
+        <Box
+          display="flex"
+          justifyContent="space-between"
+          gap={6}
+          width="100%"
+          flexDirection={isMobile ? 'column' : 'row'}
```

Comment:
consider using tailwind for easy responsive. Like `className="flex-column sm:flex-row"`